clients/vscode-hlasmplugin/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the HLASM Language Support extension are documented in this file.
 
+## [0.10.1] - 2020-04-14
+
+#### Fixed
+- Plugin crashing when writing macro instructions with operands not properly enclosed in parentheses.
```

Comment:
Was the plugin really crashing or just highligting was wrong?

Reply:
When you found the bug, you had only highlighting issues. For me, the bug caused only crash.